cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -0,0 +1,79 @@
+# Cluster Autoscaler on AWS 
+The cluster autoscaler on aws scales worker nodes within an autoscaling group. It will run as a `Deployment` in your cluster. This README will go over some of the necessary steps required to get the cluster autoscaler up and running. 
+
+## Kubernetes Version 
+Cluster autoscaler must run on v1.3.0 or greater. 
+
+## Permissions 
+You'll want to give worker nodes in your cluster access to these resources and actions, preferrably using IAM roles: 
```

Comment:
Should we note that only the nodes running the CA needs these permissions?

I understand that for convenience we add the same IAM permissions to all our workers, but I'm wondering whether we should avoid implying that this specific approach is required.